doc/data/ruff.toml

```
@@ -6,8 +6,9 @@ extend-exclude = [
     "messages/d/duplicate-argument-name/bad.py",
     "messages/s/syntax-error/bad.py",
     # syntax error in newer python versions
+    "messages/b/bare-name-capture-pattern/bad.py",  # python 3.10
```

```suggestion
    "messages/b/bare-name-capture-pattern/bad.py",
```
We dropped support for Python 3.9 in main already.